Source/g_game.c

```diff
@@ -1711,7 +1711,6 @@ static void G_DoLoadGame(void)
 
   // killough 2/14/98: load compatibility mode
   compatibility = *save_p++;
-  demo_version = complevel;
```

Comment:
Oversight?

Reply:
Probably, I can't remember why I added this line.